core/templates/domain/exploration/ExplorationObjectFactorySpec.ts

```diff
@@ -343,7 +343,7 @@ describe('Exploration object factory', () => {
 
       const firstState = exploration.getState('first state');
 
-      firstState.interaction.id = null;
+      firstState.interaction.id = '';
```

Comment:
This is not OK, the empty string is not a valid interaction ID.

Reply:
So maybe this interationId should be null?

> This is not OK, the empty string is not a valid interaction ID.

Reply:
Yes, null would be correct here if the state has no interaction specified.

Reply:
done, change back to null.